software/webapp/src/test/java/brooklyn/test/entity/TestJavaWebAppEntity.java

```diff
@@ -38,7 +39,7 @@
  * Mock web application server entity for testing.
  */
 @ImplementedBy(TestJavaWebAppEntityImpl.class)
-public interface TestJavaWebAppEntity extends VanillaJavaApp, WebAppService {
+public interface TestJavaWebAppEntity extends VanillaJavaApp, WebAppService, EntityLocal {
```

Comment:
Makes tests a little simpler, because don't need to cast to `EntityLocal` for methods like `setAttribute(...)`. It would be nice to get rid of `EntityLocal` entirely!